examples/calculator/test/calc_expression_test.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
```

Comment:
You should add this package to the list of packages to test in `.../travis/test.sh` before checkin.